torch/export/_trace.py

```diff
@@ -559,7 +551,8 @@ def _export_to_torch_ir(
             ), _ignore_backend_decomps():
                 gm_torch_level, _ = torch._dynamo.export(
                     f,
-                    dynamic_shapes=transformed_dynamic_shapes,  # type: ignore[arg-type]
+                    dynamic_shapes=dynamic_shapes,  # type: ignore[arg-type]
+                    assume_static_by_default=True,
```

Comment:
What does this buy?

Reply:
Nothing functionality wise, we just don't have to maintain this specs inverting which is already pretty confusing to me today